authorization/src/test/kotlin/com/ritense/authorization/web/rest/PermissionResourceSecurityIT.kt

```diff
@@ -61,7 +61,6 @@ class PermissionResourceSecurityIT : SecuritySpecificEndpointIntegrationTest() {
             )
         )
 
-        permissionRepository.deleteAll()
```

Comment:
We might want to restore this for saving collections where a permission was deleted.